app/assets/stylesheets/content/_tooltips.sass

```diff
@@ -26,27 +26,160 @@
 // See doc/COPYRIGHT.rdoc for more details.
 //++
 
-/***** Tooltips *****
+//***** Tooltips *****
 
-.tooltip
+// tooltips used for arbitrary elements
+
+$tooltipBackground: #e3f5ff
+$tooltipBorder: none
+$tooltipHeight: 22px
+$tooltipTextColor: $body-font-color
+$tooltipArrowSize: 6px
+
+%top-bottom
```

Comment:
@floriank The placeholder selector `%top-bottom` is not scoped to this file globally – since the name is pretty generic, I'd suggest adding the `tooltip--` prefix to prevent naming collisions.